lib/src/countly_flutter.dart

```diff
@@ -1056,6 +1107,7 @@ class Countly {
     return result;
   }
 
+  /// returns the error or success message
```

Comment:
And also you can add more word to those functions. They return error or success message but what do they do?

Reply:
They're deprecated. That's why I simply commented on the return value only.

@ArtursKadikis should I also add comments to deprecated functions.

Reply:
yup, let us also add messages to deprecated calls because they still will be used.
all public calls sohuld be well documented, even if they will be removed soon